armadillo/src/main/resources/logback-file.xml

```diff
@@ -10,6 +12,19 @@
         </layout>
     </appender>
 
+    <appender name="FILE-STDOUT" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${stdoutFile}</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <!-- daily rollover -->
+            <fileNamePattern>${stdoutFile}.%d{yyyy-MM-dd}.log</fileNamePattern>
+            <!-- keep 30 days' worth of history -->
+            <maxHistory>30</maxHistory>
+        </rollingPolicy>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread|%.8X{sessionID}] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
```

Comment:
new log file needs log file location

Do we the rollover or does ops this?

Reply:
Discussed with @DickPostma ... log rotate is not our task ... so just log the file to the configuration path